index.html

```diff
@@ -2435,34 +2440,222 @@ <h2>
+<code>true</code> for success and <code>false</code> for failure.
+        </li>
+      <li>
+<code>document</code>: a document conforming to the [[VC-DATA-MODEL-2.0]]
+        </li>
+      <li>
+<code>mediaType</code>: <code>vc+ld+json</code> or <code>vp+ld+json</code>
+        </li>
+    </ul>
+    <section>
+      <h3>Algorithm for verifying a credential or presentation secured with JOSE</h3>
+      <p>
+      The inputs for this algorithm are:
+      </p>
+      <ul>
+        <li>
+<code>inputMediaType</code>: <code>vc+ld+json+jwt</code> or
+<code>vp+ld+json+jwt</code>
+        </li>
+        <li>
+<code>inputDocument</code>: the verifiable credential secured as a JWT [[RFC7519]]
+        </li>
+      </ul>
+      <p>
+Upon receipt of the verifiable credential or presentation secured as a JWT
+[[RFC7519]], the holder or verifier follows this algorithm:
+      </p>
+      <ol>
+        <li>
+Follow the algorithm defined in [[[RFC7519]]] for Validating a JWT.
+        </li>
+        <li>
+If processing completes successfully:
+        <ol>
+          <li>
+Set <code>status</code> to <code>true</code>
+          </li>
+          <li>
+Set <code>mediaType</code> to <code>vc+ld+json</code> or <code>vp+json+ld</code>
+          </li>
+          <li>
+Set <code>document</code> to the decoded JWS payload.
+          </li>
+          <li>
+Return
+          </li>
+        </ol>
+        </li>
+        <li>
+If processing aborts for any reason or the JWT is rejected:
+        <ol>
+          <li>
+Set <code>status</code> to <code>false</code>
+          </li>
+          <li>
+Set <code>document</code> to <code>null</code>
+          </li>
+          <li>
+Set <code>mediaType</code> to <code>null</code>
+          </li>
+          <li>
+Return
+          </li>
+        </ol>
+        </li>
+        </li>
+      </ol>
+    </section>
+    <section>
+      <h3>Algorithm for verifying a credential or presentation secured with SD-JWT</h3>
+      <p>
+The inputs for this algorithm are:
+      </p>
+      <ul>
+        <li>
+<code>inputMediaType</code>: <code>vc+ld+json+sd-jwt</code>
+          </li>
+        <li>
+<code>inputDocument</code>: the verifiable credential secured with [[SD-JWT]]
+          </li>
+      </ul>
+      <p>
+Upon receipt of the verifiable credential or presentation secured with
+[[SD-JWT]], the holder or verifier follows this algorithm:
+      </p>
+        <ol>
+          <li>
+Follow the algorithm defined in [[[SD-JWT]]] for Verification of the SD-JWT.
+          </li>
+          <li>
+If processing completes successfully:
+            <ol>
+              <li>
+Set <code>status</code> to <code>true</code>
+              </li>
+              <li>
+Set <code>mediaType</code> to <code>vc+ld+json</code>
+              </li>
+              <li>
+Convert the SD-JWT payload back into the JSON claim set by reversing the process
+described in [[[SD-JWT]]] [[SD-JWT]]. Set <code>document</code> to the JSON claim set.
```

Comment:
I cannot figure out what section of [[SD-JWT]] describes the process that is supposed to be reversed. I think this reference should converted be a section-specific `data-cite`.

Reply:
Answering a question which has now been deleted, because others may share it.

Triple brackets show the full title of the referenced document, and link directly there; e.g., ReSpec turns 
`[[[SD-JWT]]]`
into
`[Selective Disclosure for JWTs (SD-JWT)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt)`
, which renders as 
[Selective Disclosure for JWTs (SD-JWT)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt).

Double brackets put a simple [cite] in the document body, which links to its citation in the references appendix, e.g., ReSpec turns 
`[[SD-JWT]]` 
in the [PR-Preview of this PR](https://pr-preview.s3.amazonaws.com/brentzundel/vc-jose-cose/pull/261.html) turns into 
`[[SD-JWT](https://pr-preview.s3.amazonaws.com/brentzundel/vc-jose-cose/pull/261.html#bib-sd-jwt)]`
, which renders as 
[[SD-JWT](https://pr-preview.s3.amazonaws.com/brentzundel/vc-jose-cose/pull/261.html#bib-sd-jwt)].

When put together, 
`[[[SD-JWT]]] [[SD-JWT]]`
, renders as 
[Selective Disclosure for JWTs (SD-JWT)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt) [[SD-JWT](https://pr-preview.s3.amazonaws.com/brentzundel/vc-jose-cose/pull/261.html#bib-sd-jwt)].

Generally speaking, `in [[cite]]` reads very badly in the body text, and just links to the appendix, while `in [[[cite]]] [[cite]]` produces a far better result in the text.

(There are many other instances of `title [[cite]]` in the existing document which I think would be better changed to `[[[cite]]] [[cite]]`.)

Reply:
The process for moving from a JSON claim set to an SD-JWT payload is a description spread across a large swath of the SD-JWT document, rather than nicely contained in a single section. I have added a link to the appendix of examples in [3ff1693](https://github.com/w3c/vc-jose-cose/pull/261/commits/3ff1693481a8df11ad5a767422a612769f04e4d9), which I hope is sufficient.